dev/workshop/clj_kondo_hooks_test.cljs

```diff
@@ -0,0 +1,52 @@
+(ns workshop.clj-kondo-hooks-test
+  (:require ["react" :as react]
+            [helix.core :refer-macros [defnc] :refer [$]]
+            [helix.dom :as d]))
+
+
+(defn comp-printer [comp] (println (str comp)))
+
+(defnc my-comp-kitchen-sink
+  "optional docstring"
+  {:meta 'data}
+  [{:keys [children]} _ref]
+  {:helix/features :fast-refresh
```

Comment:
a nitpick, doesn't look like this would affect tests: the value of `:helix/features` ought to be a map, e.g.
```suggestion
  {:helix/features {:fast-refresh true}
```